### PR TITLE
allow configuring postgres image

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: .github/workflows/foreman_plugin.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ jobs:
     runs-on: ubuntu-latest
     services:
       postgres:
-        image: 'postgres:${{ matrix.postgresql }}'
+        image: 'ghcr.io/evgeni/postgresql-evr:${{ matrix.postgresql }}'
```
Comment (ekohl) [inline .github/workflows/foreman_plugin.yml]: Can we make this an input? Something like:
```suggestion
        image: '${{ inputs.postgresql_image_name }}:${{ matrix.postgresql }}'
```

Comment (evgeni) [reply .github/workflows/foreman_plugin.yml]: like that?

File: .github/workflows/foreman_plugin.yml [hunk 2]
```diff
@@ -23,6 +23,11 @@ on:
         default: '[]'
         required: false
         type: string
+      postgresql_container:
+        description: PostgreSQL container to use
+        default: 'ghcr.io/evgeni/postgresql-evr'
```
Comment (ekohl) [inline .github/workflows/foreman_plugin.yml]:
```suggestion
        default: 'postgres'
```

**Review by ekohl — COMMENTED** [review]
Let's merge it as just a param for now.

**Comment by evgeni** [comment]
@ekohl made it a param, please merge :)

**Review by ekohl — APPROVED** [review]

**Pull request merged**